### DISCUSSION TITLE
Quarto Pub Page not found

### DISCUSSION THREAD

**Discussion by robertwelz** [quarto-dev/quarto-cli#3362] [Q&A]
Hi,

i wanted to deploy a website (created with quarto) using QuartoPub using the `quarto publish quarto-pub.` command. I believe i followed all steps in the documentation carefully, but when i publish the website, it only shows a "Page not found" error. I even deleted the old project and started a new website-project from scratch but without success. Here is the link to the website: 

[https://robertwelz.quarto.pub/mockwebsite_statistics/](https://robertwelz.quarto.pub/mockwebsite_statistics/)

If i can provide more information to solve this problem, please let me know. I'm a total beginner in this.

Thanks
Robert

**Comment by cscheid** [comment]
Can you provide the quarto website (or .qmd file) and the output of the shell commands you've used?

**Reply by robertwelz** [reply]
I uploaded all the files here: [https://github.com/robertwelz/mockwebsite_statistics.git](https://github.com/robertwelz/mockwebsite_statistics.git)

And here is the terminal output:

```
[✓] Creating quarto-pub site

Rendering for publish:

[1/5] basics-recode.qmd
[2/5] index.qmd


processing file: index.qmd
  |.....                                                                 |   7%
  ordinary text without R code

  |.........                                                             |  13%
label: setup (with options) 
List of 1
 $ include: logi FALSE

  |..............                                                        |  20%
  ordinary text without R code

  |...................                                                   |  27%
label: install-packages (with options) 
List of 1
 $ eval: logi FALSE

  |.......................                                               |  33%
  ordinary text without R code

  |............................                                          |  40%
label: load-packages (with options) 
List of 1
 $ eval: logi FALSE

  |.................................                                     |  47%
  ordinary text without R code

  |.....................................                                 |  53%
label: simple-addition
  |..........................................                            |  60%
  ordinary text without R code

  |...............................................                       |  67%
label: scatterplot
  |...................................................                   |  73%
  ordinary text without R code

  |........................................................              |  80%
label: barchart
  |.............................................................         |  87%
  ordinary text without R code

  |.................................................................     |  93%
label: excercise-2 (with options) 
List of 1
 $ eval: logi FALSE

  |......................................................................| 100%
  ordinary text without R code


output file: index.knit.md

[3/5] basics-streuung.qmd
[4/5] basics-regress.qmd
[5/5] basics-lage.qmd

[✓] Preparing to publish site
[✓] Uploading files (complete)
[✓] Deploying published site
[✓] Published site: https://quartopub.com/sites/robertwelz/mockwebsite_statistics
[✓] Account site updated: https://robertwelz.quarto.pub

```


Thanks!

**Reply by cscheid** [reply]
This is strange - I took your example, compiled it myself, and [published it](https://quartopub.com/sites/carlos-scheidegger/mockwebsite_statistics).

I wonder if something went wrong with netlify (the web publisher used at the quarto.pub backend). Does this happen consistently if you try to republish your site?

**Reply by robertwelz** [reply]
Yes. I tried different projects and with the RStudio terminal as well as with the mac terminal.

**Reply by cscheid** [reply]
I created an issue to track the bug https://github.com/quarto-dev/quarto-cli/issues/3365

**Reply by softwarenerd** [reply]
I am investigating this. I have a few other things going on today, but I will get back to you as soon as I can.

**Comment by frankcsliu** [comment]
I still see the same issue in 2026. Quite upset about this unstable feature.  Hopefully, this problem can be solved, as a new user has little to do with [the suggestions](https://answers.netlify.com/t/support-guide-i-ve-deployed-my-site-but-i-still-see-page-not-found/125) in the error message.
